confluent_platform/datadog_checks/confluent_platform/data/metrics.yaml

```diff
@@ -1600,11 +1600,15 @@ jmx_metrics:
         
 
   # Components Metrics:
-  #   - Confluent Streams Thread: https://docs.confluent.io/current/streams/monitoring.html#built-in-metrics
+  #   - Confluent Streams Thread: https://docs.confluent.io/current/streams/monitoring.html#thread-metrics
   - include:
       domain: kafka.streams
-      bean_regex: kafka\.streams:type=stream-metrics,client-id=.*
+      bean_regex: kafka\.streams:type=stream-thread-metrics,thread-id=.*
```

Comment:
Can we add this entry as new one for this bean regex instead of overwriting the current one? If we overwrite this, it might break some backwards compatibility with at least version 5 of confluent platform:
https://docs.confluent.io/legacy/platform/5.0.0/streams/monitoring.html#:~:text=MBean%3A%20kafka.streams%3Atype%3Dstream%2Dmetrics%2Cthread.client%2Did%3D%5BthreadId%5D

Moving this to a new entry will ensure both legacy and new versions to submit this metric.

Reply:
@steveny91 Thank you your advice.
Metrics have been added for compatibility up to version 5.4.10.
Please confirm.